libs/payments/stripe/src/lib/stripe.manager.spec.ts

```diff
@@ -141,4 +141,38 @@ describe('StripeManager', () => {
       expect(result).toEqual(true);
     });
   });
+
+  describe('addTaxIdToCustomer', () => {
+    it('updates customer record with incoming tax id', async () => {
+      const mockCustomer = CustomerFactory();
+      const mockUpdatedCustomer = CustomerFactory({
+        invoice_settings: {
+          custom_fields: [{ name: 'Tax ID', value: 'EU1234' }],
+          default_payment_method: null,
+          footer: null,
+          rendering_options: null,
+        },
+      });
+
+      mockClient.fetchCustomer = jest.fn().mockResolvedValueOnce(mockCustomer);
+
+      expect(mockCustomer.invoice_settings).toEqual({
+        custom_fields: null,
+        default_payment_method: 'pm',
+        footer: null,
+        rendering_options: null,
+      });
```

Comment:
This `expect` doesn't seem to test the code under test at all, but rather the factory